testtools.go

```diff
@@ -1904,8 +1906,10 @@ func (c *Kubernetes) Start(t *testing.T, now int64, i int) error {
 		chmod +x /usr/local/bin/zumount &&
 		for POOL in $(zpool list -H | cut -f 1 | grep %d); do
 			zpool destroy -f $POOL || (zumount $POOL && zpool destroy -f $POOL)
-		done`,
+		done &&
+    umount %s`,
```

Comment:
indentation, also will this have `#HOSTNAME#` interpolated into it in this context?